backend/backend.go

```diff
@@ -27,6 +27,7 @@ import (
 type Backend interface {
```

Comment:
we'll have to eventually update the comment above this.

Reply:
you mostly addressed the comment for the Backend interface in other PRs, right?